spec/amp-versioning-policy.md

```diff
@@ -1,61 +1,74 @@
+The AMP JS library is distributed via its major version only. The major version is expected to change rarely if ever (on the order of less than once per year).
+
+### AMP extensions
+
+Each AMP extension is versioned independently. Publishers of AMP documents can choose the major and minor version of an extension; they cannot choose the patch version. 
+
+- Minor versions may be changed for breaking behavior changes *without* breaking API changes.
+- Major versions *must* be changed for any breaking API changes.
 
 ## Breaking changes
 
-Breaking changes are:
+Breaking changes must be implemented in a new major or minor version of an extension or (more rarely) the AMP JS library, except in rare cases<sup>1</sup>. 
+
+Breaking changes are defined as:
 
-- Changes to the semantics of supported attributes and other exposed APIs.
+- Changes to exposed API semantics, e.g. supported attributes in an AMP extension.
 - Changes to CSS included in AMP that are not backward compatible.
 - Changes to the DOM structure (elements, attributes and attribute values) generated by AMP extensions that are not backward compatible.
+- Changes to AMP validation rules that are not backward compatible.
 
-The following special cases are not considered breaking changes:
+The following are **not** considered breaking changes:
 
 - Changes to elements and their children including their CSS if the element name starts with `i-`.
-- Changes to attribute and class names starting with `-`. But this doesn't mean, CSS backward compatibility may be broken with this mechanism, where it would otherwise be considered a breaking change.
-- Changes required to maintain the security of AMP pages.
-- Changes that break fewer than 0.1% of crawler accessible AMP pages.
+- Changes to attribute and class names starting with `i-` (but this doesn't mean that CSS backward compatibility may be broken with this mechanism, which would be considered a breaking change).
 
-For breaking changes the major version of an extension (or more rarely) the main AMP JS library should be increased.
+<sup>1</sup>The following exceptions may be implemented without a new version: 
 
-## Deprecation process
+- Changes required to maintain the security of AMP pages.
+- Changes that affect fewer than 0.1% of crawler-accessible AMP pages.
+- Changes that follow the deprecations process below.
 
-In rare cases the AMP project may decide to deprecate existing functionality. Such changes must be publicly discussed and provide significant user benefit that justifies additional work for page developers.
+## Deprecations
 
-The process is
+In rare cases, the AMP project may decide that an existing feature or API must be removed. Such breaking changes must follow the AMP deprecations process prior to removal. Deprecations must be publicly discussed and provide significant user benefit that justifies additional work for page developers.
```

Comment:
Please clarify that features should never be deprecated before the replacement has been available for at least N weeks (N should be bigger than 2)